src/utils/webrtc/webrtc.js

```diff
@@ -995,6 +956,42 @@ export default function initWebRTC(signaling, _callParticipantCollection, _local
 		sendDataChannelToAll('status', 'videoOff')
 	})
 
+	// Send the nick changed event via data channel and signaling
+	webrtc.on('nickChanged', function(name) {
+		let payload
+		if (signaling.settings.userId === null) {
+			payload = name
+		} else {
+			payload = {
+				'name': name,
+				'userid': signaling.settings.userId,
+			}
+		}
+
+		sendDataChannelToAll('status', 'nickChanged', payload)
+
+		// "webrtc.sendToAll" can not be used, as it only sends the signaling
+		// message to participants for which there is a Peer object, so the
+		// message may not be sent to participants without audio and video.
+		for (const sessionId in usersInCallMapping) {
```

Comment:
is this something that we could put into a separate function to make it clearer ? or is this very specific to the nick use case ? (could be something for later)

Reply:
Yes, I will address that when I fix the signaling messages for participants without Peer objects. So something for later :-)